### ISSUE TITLE
EE3.0.2 ee stack install --all issue

### ISSUE THREAD

**Issue by ddelaey** [rtCamp/easyengine#435]
I made a fresh install on Debian Wheezy.
When executing command:
ee stack install --all
I get:
E:Unable to correct problems, you have held broken packages.

What's wrong? How to fix it?

IP = 185.53.169.64
gau1991 public key added

root@vm1855:~# lsb_release -a
No LSB modules are available.
Distributor ID: Debian
Description: Debian GNU/Linux 7.8 (wheezy)
Release: 7.8
Codename: wheezy

root@vm1855:~# cat /proc/version
Linux version 3.2.0-4-amd64 (debian-kernel@lists.debian.org) (gcc version 4.6.3 (Debian 4.6.3-14) ) #1 SMP Debian 3.2.60-1+deb7u1

root@vm1855:~# ee stack install --all
System RAM is less than 1GB
Mail scanner packages are not going to install automatically
Adding repository for MySQL
Adding repository for Nginx
Adding repository for PHP
E:Unable to correct problems, you have held broken packages.

root@vm1855:~# locale
LANG=en_US.UTF-8
LANGUAGE=en_US.UTF-8
LC_CTYPE="en_US.UTF-8"
LC_NUMERIC="en_US.UTF-8"
LC_TIME="en_US.UTF-8"
LC_COLLATE="en_US.UTF-8"
LC_MONETARY="en_US.UTF-8"
LC_MESSAGES="en_US.UTF-8"
LC_PAPER="en_US.UTF-8"
LC_NAME="en_US.UTF-8"
LC_ADDRESS="en_US.UTF-8"
LC_TELEPHONE="en_US.UTF-8"
LC_MEASUREMENT="en_US.UTF-8"
LC_IDENTIFICATION="en_US.UTF-8"
LC_ALL=en_US.UTF-8

**Comment by MiteshShah** [comment]
@ddelaey Its seems like your previous package installation is not successfull.

can you run below command  and send us output

```
sudo apt-get clean
sudo apt-get clean all
sudo apt-get update
sudo apt-get upgrade
```

**Comment by ddelaey** [comment]
@MiteshShah

Doesn't help:

root@vm1855:~# sudo apt-get clean
root@vm1855:~# sudo apt-get clean all
root@vm1855:~# sudo apt-get update
Hit http://packages.dotdeb.org wheezy Release.gpg
Hit http://packages.dotdeb.org wheezy-php55 Release.gpg                                                            
Hit http://packages.dotdeb.org wheezy Release                                                                      
Hit http://packages.dotdeb.org wheezy-php55 Release                                                                
Hit http://packages.dotdeb.org wheezy/all amd64 Packages                                                           
Hit http://packages.dotdeb.org wheezy-php55/all amd64 Packages                                                     
Hit http://security.debian.org wheezy/updates Release.gpg                                                      
Hit http://ftp.us.debian.org wheezy Release.gpg                                             
Get:1 http://repo.percona.com wheezy Release.gpg [198 B]              
Hit http://security.debian.org wheezy/updates Release                                                  
Ign http://packages.dotdeb.org wheezy/all Translation-en                                    
Ign http://packages.dotdeb.org wheezy-php55/all Translation-en        
Hit http://ftp.us.debian.org wheezy Release                           
Get:2 http://repo.percona.com wheezy Release [15.1 kB]                
Hit http://security.debian.org wheezy/updates/main amd64 Packages               
Hit http://ftp.us.debian.org wheezy/main amd64 Packages                                     
Hit http://security.debian.org wheezy/updates/main Translation-en                        
Hit http://repo.percona.com wheezy/main amd64 Packages                                   
Hit http://ftp.us.debian.org wheezy/main Translation-en            
Ign http://repo.percona.com wheezy/main Translation-en
Fetched 15.3 kB in 3s (4955 B/s)             
Reading package lists... Done
root@vm1855:~# sudo apt-get upgrade
Reading package lists... Done
Building dependency tree       
Reading state information... Done
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
root@vm1855:~# ee stack install --all
System RAM is less than 1GB
Mail scanner packages are not going to install automatically
Adding repository for MySQL
Adding repository for Nginx
Adding repository for PHP
E:Unable to correct problems, you have held broken packages.

**Comment by harshadyeola** [comment]
@ddelaey 

try this 
```ee stack install --web  && ee stack install --all ```

**Comment by gau1991** [comment]
We fixed this issue in EasyEngine 3.1.5
